### PR TITLE
refactor(dev-infra): PRs need initial triage

### PR DESCRIPTION
**Pull request by atscott**
- The current initial triage does not include PRs. This includes them by removing the issue filter
- The merge assistance label is often applied to PRs that do not have
status=success. Caretaker should handle these as well

### CODE REVIEW

**Review by jelbourn — APPROVED** [review]
LGTM

Reviewed-for: global-approvers

**Review by gkalpak — APPROVED** [review]

File: .ng-dev/caretaker.ts [hunk 1]
```diff
@@ -9,11 +9,11 @@ export const caretaker: CaretakerConfig = {
     },
     {
       name: 'Merge Assistance Queue',
-      query: `is:pr is:open status:success label:"action: merge-assistance"`,
+      query: `is:pr is:open label:"action: merge-assistance"`,
```
Comment (josephperrott) [inline .ng-dev/caretaker.ts:12]: Was this change intentional?  Can you include it in the commit message information.

Comment (atscott) [reply .ng-dev/caretaker.ts:12]: Yea, it was intentional and it's in the commit message (the description wasn't updated though):
```
- The current initial triage does not include PRs. This includes them by removing the issue filter
- The merge assistance label is often applied to PRs that do not have
status=success. Caretaker should handle these as well
```

Comment (josephperrott) [reply .ng-dev/caretaker.ts:12]: Awesome, thanks!

**Review by josephperrott — APPROVED** [review]
LGTM
